### DISCUSSION TITLE
Apply Airflow Reverse Proxy Through the Values.yaml

### DISCUSSION THREAD

**Discussion by cjay2067** [apache/airflow#37824] [Q&A]
after the deployment of the Airflow application on Kubernetes, a manual ingress was created to allow external access to the application. However, I am unable to connect with Airflow UI. Is there a values.yaml file parameter that specifies how the reverse proxy should be implemented at the airflow application level? or how to apply it?
![MicrosoftTeams-image 22_31](https://github.com/apache/airflow/assets/110012375/cfc2c532-04d9-4acb-a8a1-cd01fec6f18e)

**Comment by shlomitubul** [comment]
@cjay2067 aside from the Ingress resources you deployed, did you install an ingress controller in your cluster? 
also, it would be helpful if you provide the error you got
